src/Tribe/Attendee_Registration/Main.php

```diff
@@ -172,7 +172,7 @@ public function get_checkout_url() {
 		 */
 		$checkout_url = apply_filters( 'tribe_tickets_attendee_registration_checkout_url', null );
 
-		if ( Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT === tribe_get_request_var( 'provider' ) ) {
+		if ( Tribe__Tickets__Commerce__PayPal__Main::ATTENDEE_OBJECT === Tribe__Tickets__Tickets::get_provider_url_slug() ) {
```

Comment:
We don't have to get rid of `tribe_get_request_var`, we need to change the parameter. `tribe_get_request_var` checks for a particular var on request: https://github.com/the-events-calendar/tribe-common/blob/master/src/functions/utils.php#L127-L150

so what we want to do here, in any case is:

```
tribe_get_request_var( Tribe__Tickets__Tickets::get_provider_url_slug() )
```